docs/misc/index.rst

```diff
@@ -2,8 +2,6 @@ Misc notes
 ==========
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-   str-builder-api
    embedding
-   protocols
```

Comment:
These two were not a real documentation, but design notes. I don't think it fits in the docs.

Reply:
I created an issue for the protocols: https://github.com/hpyproject/hpy/issues/400, and added a comment to the existing str builder issue: https://github.com/hpyproject/hpy/issues/214

Reply:
I saw this comment only after my review. Maybe we could move the string stuff to the wiki?